### PR TITLE
Hack'a'volt: Hack the app!

### PR DESCRIPTION
**Pull request by tonypls**
### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
What is the policy for the hack PRs, should I add all the feedback about what I and Alex talked about or is it more of a showcase PR? 😅

**Comment by Alportan** [comment]
> What is the policy for the hack PRs, should I add all the feedback about what I and Alex talked about or is it more of a showcase PR? 😅

I think it's fine to add your feedback as well. Also add the things you mentioned to me as well here 🎉 It will help to not miss anything. 

The hack process is pretty open so I think it should be fine to capture some things here as well 🙌  Could also be for future iterations.

**Comment by tonypls** [comment]
> What is the policy for the hack PRs, should I add all the feedback about what I and Alex talked about or is it more of a showcase PR? 😅

I think you should join our team and open PRs to this branch 😆

**Comment by VIKTORVAV99** [comment]
Hehe I'll drop some feedback and open some PRs if I have the time 🫡

File: web/src/features/panels/LeftPanel.tsx [hunk 1]
```diff
@@ -86,6 +86,20 @@ function CollapseButton({ isCollapsed, onCollapse }: CollapseButtonProps) {
...
+function ShareButton() {
+  return (
+    <button
+      data-test-id="share-button"
+      className={
+        'absolute left-full top-3 z-10 flex h-12 w-10 cursor-pointer items-center justify-center rounded-r-xl bg-zinc-50 shadow-[6px_2px_10px_-3px_rgba(0,0,0,0.1)] hover:bg-zinc-100 dark:bg-gray-900 dark:text-gray-400 dark:hover:bg-gray-800'
+      }
+      aria-label={'aria.label.showSidePanel'}
+    >
+      {<Share2Icon />}
```
Comment (VIKTORVAV99) [inline web/src/features/panels/LeftPanel.tsx:98]: This icon should switch between Apple (iOS and Safari) and non-apple devices, apple should use `<Share>` and non-apple things should use `<Share2>`.

(Drop the Icon part of the import, it's an alias version to the Share2 svg).

The devil is in the details 😈

File: web/src/features/panels/LeftPanel.tsx [hunk 2]
```diff
@@ -111,7 +125,10 @@ function OuterPanel({ children }: { children: React.ReactNode }) {
     >
       {isMobile && <MobileHeader />}
       <section className="h-full w-full">{children}</section>
-      <CollapseButton isCollapsed={!isOpen} onCollapse={onCollapse} />
+      <div className="left-full top-2 flex flex-col space-y-20">
```
Comment (VIKTORVAV99) [inline web/src/features/panels/LeftPanel.tsx:128]:
```suggestion
      <div className="left-full top-2 flex flex-col gap-20">
```
This should use gap, the native way to add space between objects in flexboxes.

**Comment by tonypls** [comment]
Closing to tidy up :)

**Pull request closed without merging**